### PR TITLE
Launch qtum wallet from Electron menu

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -104,6 +106,39 @@ function initApp() {
...
+  });
+}
+
+function showLaunchQtumWalletDialog() {
+  app.focus();
+  dialog.showMessageBox({
+    type: 'question',
+    buttons: ['Cancel', 'Launch'],
+    title: 'Launch Qtum Wallet?',
+    message: 'You are about to launch the Qtum Wallet. Bodhi will need to be shutdown to run Qtum Wallet. To run Bodhi again, you will have to close the Qtum Wallet and run Bodhi again.\n\nBodhi will shutdown shortly. Are you sure to want to continue?',
```
Comment (frankobe) [inline main.js]: localization seems necessary on electron side now

Comment (dwalintukan) [reply main.js]: true. we have quite a lot of strings for this now. let me look at possible solutions.

File: src/start_wallet.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const flags = ['-logevents'];
+  if (!isMainnet()) {
+    flags.push('-testnet');
+  }
+
+  // Start qtum-qt
+  const qtumPath = Utils.getQtumPath(false);
+  logger.debug(`qtum-qt dir: ${qtumPath}`);
+
+  const qtProcess = spawn(qtumPath, flags);
```
[the remark this answers is not in the document]
Comment (dwalintukan) [reply src/start_wallet.js]: 👍 testing a build now

File: src/utils/utils.js [hunk 3]
```diff
@@ -8,7 +8,88 @@ const { version } = require('../../package.json');
...
+    default: {
+      throw new Error('Operating system not supported');
+    }
+  }
+
+  return path.replace('app.asar', 'app.asar.unpacked');
+}
+
 class Utils {
+  static getQtumPath(forDaemon) {
```
[the remark this answers is not in the document]
Comment (dwalintukan) [reply src/utils/utils.js]: changed to use constants instead of bool

**Pull request merged**